### PR TITLE
Handle presentationType in Homepage's Presentation list

### PR DESCRIPTION
**Pull request by fjvallarino**
Originally this PR was for fixing an annoyance regarding right clicking on Presentation List, but then I realized I had not considered the Homepage also has a list of Presentations. This fixes both situations.

The original PR comment was: _This just restores the ability of opening presentations in new tabs then right-clicking or ctrl-clicking (it was disabled by the previous commit). It's available for testing at:_ https://apps-stage-1.risevision.com/?cid=a847f5bb-e61a-4925-817a-552075edf203

@Rise-Vision/apps @santiagonoguez @stulees please review

### CODE REVIEW

File: web/scripts/editor/controllers/ctr-presentation-list.js [hunk 1]
```diff
@@ -36,11 +36,11 @@ angular.module('risevision.editor.controllers')
...
       });
 
-      $scope.openPresentation = function(presentation) {
+      $scope.getEditorLink = function(presentation) {
         if (presentation.presentationType !== HTML_PRESENTATION_TYPE) {
-          $state.go('apps.editor.workspace.artboard', { presentationId: presentation.id });
+          return 'apps.editor.workspace.artboard({ presentationId: presentation.id })';
         } else {
-          $state.go('apps.editor.templates.edit', { presentationId: presentation.id });
+          return 'apps.editor.templates.edit({ presentationId: presentation.id })';
```
Comment (alex-deaconu) [inline web/scripts/editor/controllers/ctr-presentation-list.js]: @fjvallarino curious, why duplicate this code in ctr-home.js? Can't this be a factory function? Thanks.

Comment (fjvallarino) [reply web/scripts/editor/controllers/ctr-presentation-list.js]: @alex-deaconu I thought about moving it somewhere with common code, but I preferred to keep it in the controller since it's view related and could not think of a good place. Given we already have view related code in services, it makes sense moving it there. I'll update

Comment (alex-deaconu) [reply web/scripts/editor/controllers/ctr-presentation-list.js]: @fjvallarino maybe a filter would work too? Thanks.

**Comment by fjvallarino** [comment]
@alex-deaconu updated

File: test/unit/launcher/controllers/ctr-home.tests.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 'use strict';
 describe('controller: Home', function() {
   beforeEach(module('risevision.apps.launcher.controllers'));
+  beforeEach(module('risevision.template-editor.services'));
+  beforeEach(module(mockTranlate()));
```
Comment (alex-deaconu) [inline test/unit/launcher/controllers/ctr-home.tests.js]: @fjvallarino are these changes still needed? Thanks.
